packages/backend/src/models/OpenIdIdentitiesModel.ts

```diff
@@ -72,19 +73,37 @@ export class OpenIdIdentityModel {
 
     async getIdentitiesByUserId(
         userId: number,
-    ): Promise<OpenIdIdentitySummary[]> {
+    ): Promise<Record<OpenIdIdentityIssuerType, OpenIdIdentitySummary[]>> {
         const identities = await this.database('openid_identities').where(
             'user_id',
             userId,
         );
+
+        const defaultIdentities = Object.values(
+            OpenIdIdentityIssuerType,
+        ).reduce(
+            (acc, curr) => ({ ...acc, [curr]: [] }),
+            {} as Record<OpenIdIdentityIssuerType, OpenIdIdentitySummary[]>,
```

Comment:
I think you can remove all the "forced" typing if you type the reduce `.reduce<Record<OpenIdIdentityIssuerType, OpenIdIdentitySummary[]>(...)`

Reply:
😮  thanks @ZeRego !

Reply:
ok, was able to remove one type casting, couldn't fix the default object one :/

Reply:
I would prefer returning all (even if some/all are empty) so we can check if they're available on the frontend - unless we want to move that check to the backend? 🤔